packages/common/src/types/document.ts

```diff
@@ -52,7 +52,7 @@ export type TDocumentFindRequest = TCollectionRequest & {
 };
 
 export type TDocumentCreateRequest = TCollectionRequest & {
-  document: TDocumentField[];
+  document: Record<string, TDocumentField>;
```

Comment:
does Record is graphql friendly? It will be painful if frontend dev get the JSON type